README.md

```diff
@@ -67,3 +67,19 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Testing
+
+This app utilises pytest for its unit and integration tests.
+
+In order to use pytest please first install it using pip and the following in the terminal
+
+```bash
+$ pip install pytest
```

Comment:
No need to touch pip and we don't want to introduce extra manual steps to run the project.

We are using Poetry to manage the project's dependencies. When you run `poetry install`, it looks at your pyproject.toml file and downloads whatever it needs to (like Flask/requests) and puts them in your virtual environment. 

You already added pytest to that list so no extra step is needed. Everyone will automatically install pytest when they run "poetry install" which they need to do anyway.

`pip install pytest` is likely installing pytest globally on your machine. If you found that necessary to get tests to run within VS Code, then that means your VS Code is using your global Python rather than the isolated Python environment created by Poetry. Let us know if you want a hand fixing that.